common/libs/transaction.js

```diff
@@ -47,8 +48,8 @@ export function getTransactionFields(tx: EthTx) {
   const [nonce, gasPrice, gasLimit, to, value, data, v, r, s] = tx.toJSON();
 
   return {
-    // No value comes back as '0x', but most things expect '0x0'
-    value: value === '0x' ? '0x0' : value,
+    // No value comes back as '0x', but most things expect '0x00'
```

Comment:
any link to this? seems like we're changing it once already, so would be good to know where this info is coming from.

Reply:
Yeah, would definitely be good to figure out the "correct" behavior here. I just used these values based on complaining libraries. `0x0` from `0x` came from the node, `0x00` came from Trezor wanting padded values.